app/controllers/interventions_controller.rb

```diff
@@ -1,15 +1,20 @@
 class InterventionsController < ApplicationController
+  include SerializeInterventionHelper
 
   before_action :authenticate_educator!
 
   def create
-    @intervention = Intervention.new(intervention_params)
-    @intervention.start_date = Time.zone.now.to_date
+    intervention = Intervention.new(intervention_params)
+    intervention.start_date = Time.zone.now.to_date
     if intervention_params[:end_date].present?
-      @intervention.end_date = Date.parse(intervention_params[:end_date])
+      intervention.end_date = Date.parse(intervention_params[:end_date])
+    end
+    
+    if intervention.save
+      render json: serialize_intervention(intervention)
+    else
+      render json: { errors: intervention.errors.full_messages }, status: 422
```

Comment:
This serializes errors back as JSON, so the JS code can parse them and display them (similar to what the `create.js.erb` files were doing before).